integrations/travisci.rst

```diff
@@ -43,7 +43,11 @@ To enable **Travis CI** support, you need to create a *.travis.yml* file and pas
     script:
     # Download dependencies and build project
       - conan install .
-      - make
+    # Call your build system
+      - cmake . -G "Unix makefiles" && make
+      - cmake --build .
```

Comment:
it is `make` in the line above, and `cmake --build .` here... remove one of them